### PR TITLE
Add logging to display the parties that are missing a flow session within AnonymisePartiesFlow

### PR DESCRIPTION
**Pull request by willhr3**

### CODE REVIEW

File: workflows/src/main/kotlin/com/r3/corda/lib/tokens/workflows/internal/flows/confidential/AnonymisePartiesFlow.kt [hunk 1]
```diff
@@ -20,7 +20,9 @@ class AnonymisePartiesFlow(
     @Suspendable
     override fun call(): Map<Party, AnonymousParty> {
         val sessionParties = sessions.map(FlowSession::counterparty)
-        require(sessionParties.containsAll(parties)) { "You must provide sessions for all parties." }
+        val partiesWithoutASession = parties.minus(sessionParties)
```
Comment (kasiastreich) [inline workflows/src/main/kotlin/com/r3/corda/lib/tokens/workflows/internal/flows/confidential/AnonymisePartiesFlow.kt]: I think that types don't match up? parties are list of Party, sessionParties are FlowSessions

Comment (willhr3) [reply workflows/src/main/kotlin/com/r3/corda/lib/tokens/workflows/internal/flows/confidential/AnonymisePartiesFlow.kt]: it maps to the FlowSession.counterparty when it's declared on line 22

Comment (kasiastreich) [reply workflows/src/main/kotlin/com/r3/corda/lib/tokens/workflows/internal/flows/confidential/AnonymisePartiesFlow.kt]: true, I haven't woken up :D

**Comment by rogersanick** [comment]
@willhr3 - LGTM! Are you still working on this? Let me know, I'm happy to rebase / merge.

**Comment by willhr3** [comment]
@rogersanick I thought it had already been merged - my bad!

**Pull request merged**